README.rst

```diff
@@ -168,6 +178,176 @@ Response
         "description": "optional description"
     }
 
+IpAddress-Subnet List and Create View
+#####################################
+An api enpoint to retrieve or create IP addresses under a specific subnet
+
+GET
+++++
+Returns the list of IP addresses under a particular subnet.
+
+.. code-block:: text
+
+    /api/v1/subnet/<subnet_id>/ip-address/
+
+POST
+++++
+Create an Ip Address record
+
+.. code-block:: text
+
+    /api/v1/subnet/<subnet_id>/ip-address/
+
+===========    ========================================
+Param          Description
+===========    ========================================
+ip_address     IPv6/IPv4 address value
+subnet         Subnet UUID
+description    Optional description for the IP address
+===========    ========================================
+
+Subnet List/Create View
+#######################
+
+An api endpoint to create or retrieve the list of subnet instances
+
+GET
+++++
+Returns the list of Subnet instances
```

Comment:
add final dot, add double quotes to ``Subnet``, leave blank line after title, the plus signs should be only 3: `+++`